packages/sdk/src/utils/chain-constants.ts

```diff
@@ -302,6 +303,24 @@ export const CONTRACT_ADDRESSES: {
     },
     l2: DEFAULT_L2_CONTRACT_ADDRESSES,
   },
+  [L2ChainID.MODE_MAINNET]: {
```

Comment:
Mode sepolia addresses can be found here: https://github.com/ethereum-optimism/superchain-registry/pull/107/files#diff-eeca29e52df49e42d3e255a7b46c5acf519012a22c85c11c31d3ca9b2478be77